### DISCUSSION TITLE
Repo architecture change

### DISCUSSION THREAD

**Discussion by MigraineMan33** [marcel-licence/esp32_basic_synth#78] [General]
Noticed that the repo structure has changed.  The local "boards" subdirectory has been eliminated.  The ML_SynthTools project contains the necessary header and board files, and should be installed in your $HOME/Arduino/libraries/ directory.  Makes sense - common hardware platform elements in one location rather than having to maintain multiple local copies in various project trees.

Marcel, sent you an email.

**Comment by marcel-licence** [comment]
Hi thank you.
Yes I put this into the library because the configuration is the same for each project.
I try to centralize stuff which will be reused.
Nothing will be lost =)
